components/nimbus/src/lib.rs

```diff
@@ -393,6 +403,27 @@ pub struct BucketConfig {
     pub total: u32,
 }
 
+// This type is passed across the FFI to client consumers, e.g. UI for testing tooling.
+pub struct AvailableExperiment {
```

Comment:
Would it be useful to include the top-level `featureIds` array from the experiment here?  I guess I'm wondering if that would be helpful for filtering or organizing experiments by feature. I guess this information is contained in the `branches` collection too...

Reply:
I figured that feature ids were about to undergo a change; as you say the feature ids are available in the branches.